usr/sbin/registercloudguest

```diff
@@ -170,6 +173,21 @@ def cleanup():
     utils.clear_new_registration_flag()
     utils.clean_framework_identifier()
 
+# ----------------------------------------------------------------------------
+def setup_registry(registration_target, user, password):
+    """
+    Run the registration process for the container registry
+    """
+    registry_fqdn = registration_target.get_registry_FQDN()
+    if not utils.is_registry_registered(registry_fqdn)
+       and not utils.setup_registry(registry_fqdn, user, password):
```

Comment:
I simplified this in #187, also pyflakes 3.6 complained about the current code being invalid syntax.